Python/errors.c

```diff
@@ -1960,20 +1960,41 @@ PyErr_ProgramText(const char *filename, int lineno)
     return res;
 }
 
+/* Function from Parser/tokenizer/file_tokenizer.c */
+extern char* _PyTokenizer_FindEncodingFilename(int, PyObject *);
+
 PyObject *
 _PyErr_ProgramDecodedTextObject(PyObject *filename, int lineno, const char* encoding)
```

Comment:
should this be `_PyErr_DecodedProgramTextObject`?

Reply:
This already was here.